### DISCUSSION TITLE
Xash not starting up

### DISCUSSION THREAD

**Discussion by DudeWithTheFood** [sultim-t/xash-rt#176] [General]
so i have a geforce gtx 1650 and when i start up xash it says (Host_InitError: Can't find physical device with ray tracing support)
Help!

**Comment by saturnotaku** [comment]
Your GPU doesn't support ray tracing so this mod isn't going to work.

**Reply by derolxd** [reply]
Since April 2019 nVidia included RayTracing support from the GTX 1060 6GB (not 3GB) onwards. It works, the problem is that it doesn't give the results that one would like. With this in mind the mod should work perfectly just with low frames.
